registry/Scripts/script.sh

```diff
@@ -126,6 +127,12 @@ fetchCredentials() {
             "${secret}?api-version=2016-10-01" -H "Authorization: Bearer ${TOKEN}" | jq -r .value)
         htpasswd -Bb .htpasswd ${SECRET_NAME} ${SECRET_VALUE}
     done
+
+    if [ ! -s .htpasswd ]; then
```

Comment:
Will this also detect if the fail does not exit?

Reply:
This is to check for empty content only.

Reply:
It will probably not be enough then